[2025-07-16T18:08:56.982+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-16T18:08:57.445+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bank_reviews_etl_and_analysis_pipeline.dbt_run_transactional scheduled__2025-06-29T00:00:00+00:00 [queued]>
[2025-07-16T18:08:57.485+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bank_reviews_etl_and_analysis_pipeline.dbt_run_transactional scheduled__2025-06-29T00:00:00+00:00 [queued]>
[2025-07-16T18:08:57.488+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-07-16T18:08:57.543+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): dbt_run_transactional> on 2025-06-29 00:00:00+00:00
[2025-07-16T18:08:57.578+0000] {standard_task_runner.py:72} INFO - Started process 1058 to run task
[2025-07-16T18:08:57.612+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'bank_reviews_etl_and_analysis_pipeline', 'dbt_run_transactional', 'scheduled__2025-06-29T00:00:00+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/dag_load_reviews.py', '--cfg-path', '/tmp/tmpivime2ic']
[2025-07-16T18:08:57.627+0000] {standard_task_runner.py:105} INFO - Job 32: Subtask dbt_run_transactional
[2025-07-16T18:08:57.820+0000] {task_command.py:467} INFO - Running <TaskInstance: bank_reviews_etl_and_analysis_pipeline.dbt_run_transactional scheduled__2025-06-29T00:00:00+00:00 [running]> on host 569ba507bc01
[2025-07-16T18:08:58.288+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohammed' AIRFLOW_CTX_DAG_ID='bank_reviews_etl_and_analysis_pipeline' AIRFLOW_CTX_TASK_ID='dbt_run_transactional' AIRFLOW_CTX_EXECUTION_DATE='2025-06-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-29T00:00:00+00:00'
[2025-07-16T18:08:58.295+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-07-16T18:08:58.301+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-07-16T18:08:58.304+0000] {logging_mixin.py:190} INFO - Current task name:dbt_run_transactional state:running start_date:2025-07-16 18:08:57.447802+00:00
[2025-07-16T18:08:58.306+0000] {logging_mixin.py:190} INFO - Dag name:bank_reviews_etl_and_analysis_pipeline and current dag run status:running
[2025-07-16T18:08:58.308+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-07-16T18:08:58.312+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-07-16T18:08:58.463+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/.dbt/dbt_projects/bank_reviews_transactional && dbt run']
[2025-07-16T18:08:58.541+0000] {subprocess.py:99} INFO - Output:
[2025-07-16T18:09:15.318+0000] {subprocess.py:106} INFO - [0m18:09:15  Running with dbt=1.10.4
[2025-07-16T18:09:16.372+0000] {subprocess.py:106} INFO - [0m18:09:16  Registered adapter: postgres=1.9.0
[2025-07-16T18:09:18.983+0000] {subprocess.py:106} INFO - [0m18:09:18  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-07-16T18:09:18.985+0000] {subprocess.py:106} INFO - There are 1 unused configuration paths:
[2025-07-16T18:09:18.987+0000] {subprocess.py:106} INFO - - models.bank_reviews_transactional.transactional
[2025-07-16T18:09:19.467+0000] {subprocess.py:106} INFO - [0m18:09:19  Found 1 model, 1 source, 433 macros
[2025-07-16T18:09:19.478+0000] {subprocess.py:106} INFO - [0m18:09:19
[2025-07-16T18:09:19.482+0000] {subprocess.py:106} INFO - [0m18:09:19  Concurrency: 4 threads (target='dev')
[2025-07-16T18:09:19.485+0000] {subprocess.py:106} INFO - [0m18:09:19
[2025-07-16T18:09:24.646+0000] {subprocess.py:106} INFO - [0m18:09:24  1 of 1 START sql table model transactional.clean_reviews ....................... [RUN]
[2025-07-18T18:05:14.548+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-18T18:05:15.372+0000] {taskinstance.py:2604} INFO - Dependencies not met for <TaskInstance: bank_reviews_etl_and_analysis_pipeline.dbt_run_transactional scheduled__2025-06-29T00:00:00+00:00 [skipped]>, dependency 'Task Instance State' FAILED: Task is in the 'skipped' state.
[2025-07-18T18:05:15.400+0000] {local_task_job_runner.py:166} INFO - Task is not able to be run
